src/pages/openoffice.js

```diff
@@ -46,34 +55,34 @@ export default function OpenOffice() {
   return (
     <div className={`${styles.main}`}>
       <div className={`${styles.openOfficeHeader}`}>
-        <h1>Open Office Calendar</h1>
+        <ComputerWindow className={styles.title}>
+          <h2>Open Office Calendar</h2>
+        </ComputerWindow>
         {week === 'week1' ? (
           <>
-            <p>You&apos;re viewing the Week 1 schedule</p>
-            <button
-              type="button"
-              className={`${styles.openOfficeWeekButton}`}
+            {/* <p>We are currently on Week {curWeek} shifts.</p> */}
+            <p>You&apos;re viewing the Week 1 schedule.</p>
+            <Button
               onClick={() => {
                 setWeek('week2');
                 setWeekNum(1);
               }}
             >
-              <p>Check out week 2</p>
-            </button>
+              <h2>Week 2</h2>
```

Comment:
i think keeping some indication that this isnt the current week we are looking at, but we can switch to it might be a better idea here (ik it wasnt in the figma but just a thought)